dashboard/test/models/resource_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ResourceTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
Usually Dave just suggestions that I add a test to make sure I can create a new resource here to start this file